fnmatch/fnmatch.py

```diff
@@ -64,6 +79,29 @@ def fnmatchcase(name, pat):
     match = _compile_pattern(pat)
     return match(name) is not None
 
+_re_special_chars_map = {i: '\\' + chr(i) for i in b'()[]{}?*+-|^$\\.&~# \t\n\r\v\f'}
+def _re_translate(p, chmap):
+    oldp = p
+    for idx,c in enumerate(p):
+     print('[%d]=%s' % (idx, c))
```

Comment:
Leftover? Also please remove the other debug statements.

Reply:
Correct.